package.json

```diff
@@ -52,6 +52,7 @@
     "prettier": "^1.7.4",
     "react": "^16.0.0",
     "react-dom": "^16.0.0",
+    "react-lifecycles-compat": "^1.1.0",
```

Comment:
This is `devDependencies`, so user apps will break unless they have this package installed on their side. I'll add an eslint rule to prevent such things

@diasbruno you may want to hold off the release until fixing that

Reply:
Sure, no problem. Let me know when it's ready. 

cc @kaushik94

Reply:
Done as a part of #642

Reply:
Yes my bad, thanks for fixing all this 👍